distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -228,39 +229,52 @@ type Filter_Condition
        Creates a Single_Choice Widget for delimiters.
     default_widget : Boolean -> Boolean -> Boolean -> Boolean -> Boolean -> Widget
     default_widget include_comparable=True include_text=True include_boolean=True include_nullable=True include_numeric=True =
-        options_before = Vector.build options_builder->
+        text = if include_text then [Option "<Text Value>" "''"] else []
+        number = if include_numeric then [Option "<Number Value>" "0"] else []
+        boolean = if include_boolean then [Option "<True/False>" "True"] else []
+
+        with_all_types = Widget.Single_Choice values=text+number+boolean display=Display.Always
+        with_number_text = Widget.Single_Choice values=text+number display=Display.Always
+
+        ## Can't use auto-scoping as we allow predicates or a Filter_Condition.
+        fqn = Meta.get_qualified_type_name Filter_Condition
+
+        options = Vector.build builder->
+            builder.append (Option "Equals" fqn+".Equal" [["to", with_all_types]])
+            builder.append (Option "Not Equals" fqn+".Not_Equal" [["to", with_all_types]])
```

Comment:
Should there not be an underscore here? Are we moving away from every you see is code? If that is the case Not Equal feels more natural to me than "Not Equals". Though Equals is definitely better than Equal